forloop_common_structures/core/pipeline.py

```diff
@@ -5,7 +5,7 @@
 @dataclass
 class Pipeline:
     name: str
-    start_node_uid: str = 0
+    start_node_uid: str = "0"
```

Comment:
Are you positive that this won't crash somewhere? @ZaiZu2

Reply:
Lets see :))))

Reply:
For what I tested, it did not. It did crash with the previous value, tho.